app/_src/guides/consumer-producer-policies.md

```diff
@@ -0,0 +1,231 @@
+  labels:
+    kuma.io/sidecar-injection: enabled" | kubectl apply -f -
+```
+
+Now we can create deployment we will be using to communicate with our demo-app from a first-consumer namespace:
+
+```shell
+kubectl run consumer --image nicolaka/netshoot -n first-consumer --command -- /bin/bash -c "ping -i 60 localhost"
+```
+
+and from the second-consumer namespace:
+
+```shell
+kubectl run consumer --image nicolaka/netshoot -n second-consumer --command -- /bin/bash -c "ping -i 60 localhost"
+```
+
+You can make now make a couple of requests to our demo app to check if everything is working: 
+
+```shell
+kubectl exec -n first-consumer consumer -- curl -s -XPOST demo-app.kuma-demo:5000/increment
+```
+
+You should see something similar to:
+
+```json
+{"counter":"1","zone":"local","err":null}
+```
+
+## Namespace scoped policies
+
+Now that we have our setup we can start playing with policies. Let's create a simple MeshTimeout policy in `kuma-demo` namespace:
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: producer-timeout
+  namespace: kuma-demo
+  labels:
+    kuma.io/mesh: default
+    kuma.io/origin: zone
+spec:
+  to:
+    - targetRef:
+        kind: MeshService
+        name: demo-app
+      default:
+        http:
+          requestTimeout: 1s" | kubectl apply -f -
+```
+
+We can now inspect applied policy labels:
+
+```shell
+kubectl get meshtimeout -n kuma-demo producer-timeout -o jsonpath='{.metadata.labels}'
+```
+
+We should see output like:
+
+```json
+{
+  "k8s.kuma.io/namespace": "kuma-demo",
+  "kuma.io/env": "kubernetes",
+  "kuma.io/mesh": "default",
+  "kuma.io/origin": "zone",
+  "kuma.io/policy-role": "producer",
+  "kuma.io/zone": "default"
+}
+```
+
+{{site.mesh_product_name}} is adding custom labels to our policy. The one that interests us most at the moment is `kuma.io/policy-role`.
+This is new label that indicates policy role. Possible values of this label are:
+
+- `system` - policies applied in system namespace
+- `workload-owner` - policies that don't specify `spec.from` and `spec.to` sections or specify only `spec.from` section
+- `consumer` - policies targeting MeshServices from different namespace or targeting MeshService by labels
+- `producer` - policies targeting MeshServices from the same namespace in `spec.to[]` section 
+
+### Producer consumer model
+
+With namespace scoped policies we've introduced producer/consumer model for policies. 
+
+**Producer** is someone who authors and operates service. Producer can create default policies that will be used when communicating with service. 
+Producer policies will be created in the same zone as MeshService they target. Producer policies will be synced to other zones. 
+
+**Consumer** is someone that utilizes a service by communicating with it. Consumer policies will be applied in consumer 
+namespace and will target MeshService from different namespace. Consumer policy will take effect only in consumer namespace.
```

Comment:
```suggestion
namespace and may target MeshService from different namespace. Consumer policy will take effect only in consumer namespace.
```

It can be in the same namespace (using labels).